.githooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Make sure /usr/local/bin is in the path
+PATH="$PATH:/usr/local/bin"
```

Comment:
This seems like something needed for possible other future hooks as well, and could be needed only for some people, meaning we could miss it in new hook files. 
Can it be extracted?

Reply:
Not sure how. That's probably the advantage of something like husky (if they do that by default).

Reply:
@daun did the PR with Husky work out of the box for you? If yes, I would propose that we just go with that PR instead of this one.